### DISCUSSION TITLE
v2.1.0 (Beta)

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#2545] [Releases]
This release brings two major features:

USB support is here for macOS *and* jailbroken iOS. You can attach a device to your VM with the new USB icon in the toolbar. You can enable USB 3.0 in the Share settings for faster devices on supported targets. Note that older versions of Windows (e.g. Windows 7 and below) require installing additional drivers for USB 3.0.

UTM SE is a new build for iOS that does not require JIT (and therefore is much slower). It can therefore be sideloaded on any iOS device that is not jailbroken. Jailbreak users are still recommended to install the standard UTM DEB package for faster JIT speeds and USB support.

## Installation
Visit [https://getutm.app/install/][1] for the most up to date installation instructions.

## Changes
* QEMU updated from v5.2.0 to v6.0.0
* USB support: attach a USB device to the VM (macOS and jailbroken iOS 14.5 and below only)
* UTM SE (Slow Edition): JITless iOS build available to sideload on *any* non-jailbroken device (thanks @ktemkin)
* Screamer audio device for PPC mac99 machines (no setting changes needed)

Please check the full list on [Github][2] and help report any bug you find that is not listed.

[1]: https://getutm.app/install/
[2]: https://github.com/utmapp/UTM/issues

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v2.1.0'>v2.1.0 (Beta)</a>.</em>

**Comment by HeySora** [comment]
Is this a Debug build? Because lots of text is now being printed on "Console Only" virtual machines (even though Debug Logging is disabled) and the performance seems to be hurt a lot because of that.

(Update: the debug prints also happen on graphic virtual machines, inside the BIOS when pressing F2)

**Reply by osy** [reply]
@conath did you build EFI in release mode?

**Reply by conath** [reply]
Unknowingly, I did indeed build them in Debug mode. I am compiling again in Release mode now, sorry about this!

**Reply by conath** [reply]
I have uploaded the re-compiled release binaries: https://github.com/utmapp/edk2/pull/1#issuecomment-846011714

@HeySora If you want to replace the BIOS immediately without waiting for UTM update:
For x86_64 VM, import the file OVMF… into your UTM VM in the Drives tab, then change type to "Disk Image" and "PC System Flash". For AARCH64 VM, you instead choose the file QEMU-AARCH64… and set its type to BIOS.

**Reply by HeySora** [reply]
@conath It's okay, do not worry!
Thank you a lot for these binaries, performance is stellar now!!

**Comment by dekefake** [comment]
Thanks a lot for this update ! Because of USB support, i've been able to use Volkswagen VCDS tools to connect with my Audi A3 3.2 V6, with an interface whose i386 driver hasn't been updated since 2013. UTM 2.10 + Windows XP worked like a charm !

<img width="1680" alt="Capture d’écran 2021-05-21 à 20 55 42" src="https://user-images.githubusercontent.com/11983192/119189275-271f3600-ba7c-11eb-8dda-73088de129d1.png">

Windows XP.qcow2 : https://mega.nz/file/0OQlUAIL#__QPIJeEynSG49YbIlQG2oQqoTC2YGWEdHi08zeIpi8

**Reply by mstaack** [reply]
how well does this run for you? does booting up xp take a long time?

**Reply by conath** [reply]
On my MacBook Pro M1 running macOS 12.1 and UTM 2.4.1, Windows XP SP3 32 bit takes 48 seconds to start up & auto-login. I used the [Windows XP “UTM Fan Edition”](https://chrisp.cafe/UTM/xp-utm-fan-edition.html) installer to set it up.

**Comment by HenkPoley** [comment]
Windows 10 ARM64 Insiders Preview doesn't seem to boot on this release (I know beta on beta).

**Reply by osy** [reply]
Thanks can you post a new issue with more information so I can properly track it?

**Reply by HenkPoley** [reply]
Created an issue here: https://github.com/utmapp/UTM/issues/2558
